# 
# Translators:
# José Macau <jose.macau@gmail.com>, 2023
# 
msgid ""
msgstr ""
"Project-Id-Version: Lizmap Web Client 3.5 dataviz\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-03-15 11:13++0100\n"
"PO-Revision-Date: 2023-11-28 15:42+0000\n"
"Last-Translator: José Macau <jose.macau@gmail.com>, 2023\n"
"Language-Team: Portuguese (https://app.transifex.com/3liz-1/teams/62012/pt/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: pt\n"
"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % 1000000 == 0 ? 1 : 2;\n"

#: dataviz~dataviz.dock.subtitle
msgctxt "dataviz~dataviz.dock.subtitle"
msgid "Dataviz"
msgstr "Gráficos"

#: dataviz~dataviz.dock.title
msgctxt "dataviz~dataviz.dock.title"
msgid "Dataviz"
msgstr "Gráficos"

#: dataviz~dataviz.log.access_denied.detail
msgctxt "dataviz~dataviz.log.access_denied.detail"
msgid ""
"You have no right to access this ressource. Have you used the right login ?"
msgstr "Sem permissão para aceder a este recurso. Utilizador correto?"

#: dataviz~dataviz.log.access_denied.title
msgctxt "dataviz~dataviz.log.access_denied.title"
msgid "Access denied"
msgstr "Acesso negado"

#: dataviz~dataviz.log.invalid_plot_type.detail
msgctxt "dataviz~dataviz.log.invalid_plot_type.detail"
msgid "The given plot type \"%s\" is invalid."
msgstr "O tipo de gráfico fornecido \"%s\" é inválido."

#: dataviz~dataviz.log.invalid_plot_type.title
msgctxt "dataviz~dataviz.log.invalid_plot_type.title"
msgid "Invalid plot type"
msgstr "Tipo de gráfico inválido"

#: dataviz~dataviz.log.layer_not_found.detail
msgctxt "dataviz~dataviz.log.layer_not_found.detail"
msgid ""
"The layer with the given ID \"%s\" cannot be found in the published QGIS "
"project."
msgstr ""
"A camada com o ID fornecido \"%s\" não pode ser encontrada no projeto QGIS "
"publicado."

#: dataviz~dataviz.log.layer_not_found.title
msgctxt "dataviz~dataviz.log.layer_not_found.title"
msgid "Layer not found"
msgstr "Camada não encontrada"

#: dataviz~dataviz.log.no_configuration.title
msgctxt "dataviz~dataviz.log.no_configuration.title"
msgid "There is no dataviz configuration for this project yet."
msgstr "Ainda não existe configuração de gráficos para este projeto."

#: dataviz~dataviz.log.no_data.detail
msgctxt "dataviz~dataviz.log.no_data.detail"
msgid "The plot source data cannot be fetched from the vector layer \"%s\"."
msgstr ""
"Os dados de origem do gráfico não podem ser obtidos a partir da camada "
"vetorial \"%s\"."

#: dataviz~dataviz.log.no_data.title
msgctxt "dataviz~dataviz.log.no_data.title"
msgid "Plot data not available"
msgstr "Dados do gráfico não disponíveis"

#: dataviz~dataviz.log.no_layers.detail
msgctxt "dataviz~dataviz.log.no_layers.detail"
msgid "The table of layers inside the dataviz configuration is empty."
msgstr ""
"A tabela de camadas dentro da configuração dos gráficos encontra-se vazia."

#: dataviz~dataviz.log.no_layers.title
msgctxt "dataviz~dataviz.log.no_layers.title"
msgid "Dataviz layers is empty"
msgstr "As camadas do gráfico encontram-se vazias"

#: dataviz~dataviz.log.plot_configuration_not_found.detail
msgctxt "dataviz~dataviz.log.plot_configuration_not_found.detail"
msgid ""
"The plot configuration cannot be found or parsed by Lizmap dataviz API."
msgstr ""
"A configuração do gráfico não pode ser encontrada ou analisada pela API "
"Lizmap dos gráficos."

#: dataviz~dataviz.log.plot_configuration_not_found.title
msgctxt "dataviz~dataviz.log.plot_configuration_not_found.title"
msgid "Plot configuration not found"
msgstr "Configuração de gráficos não encontrada"

#: dataviz~dataviz.log.project_not_found.detail
msgctxt "dataviz~dataviz.log.project_not_found.detail"
msgid ""
"The QGIS project \"%s.qgs\" does not exist or is not available in the "
"repository \"%s\"."
msgstr ""
"O projeto QGIS \"%s.qgs\" não existe ou não está disponível no repositório "
"\"%s\"."

#: dataviz~dataviz.log.project_not_found.title
msgctxt "dataviz~dataviz.log.project_not_found.title"
msgid "Lizmap project not found"
msgstr "Projeto Lizmap não encontrado"

#: dataviz~dataviz.log.repository_not_found.detail
msgctxt "dataviz~dataviz.log.repository_not_found.detail"
msgid "The Lizmap repository \"%s\" does not exist or is not available."
msgstr "O repositório Lizmap \"%s\" não existe ou não se encontra disponível."

#: dataviz~dataviz.log.repository_not_found.title
msgctxt "dataviz~dataviz.log.repository_not_found.title"
msgid "Lizmap repository not found"
msgstr "Repositório Lizmap não encontrado"

#: dataviz~dataviz.log.request_not_supported.detail
msgctxt "dataviz~dataviz.log.request_not_supported.detail"
msgid "The given request \"%s\" is not supported by Lizmap dataviz API."
msgstr "O pedido efetuado \"%s\" não é suportado pela API Lizmap dos gráficos."

#: dataviz~dataviz.log.request_not_supported.title
msgctxt "dataviz~dataviz.log.request_not_supported.title"
msgid "Request is not supported"
msgstr "Pedido não suportado"

#: dataviz~dataviz.log.wrong_credentials.detail
msgctxt "dataviz~dataviz.log.wrong_credentials.detail"
msgid "The given login and password are incorrect. Cannot authenticate."
msgstr ""
"O utilizador e a palavra-passe fornecidos estão incorretos. Não é possível "
"autenticar."

#: dataviz~dataviz.log.wrong_credentials.title
msgctxt "dataviz~dataviz.log.wrong_credentials.title"
msgid "Wrong credentials"
msgstr "Credenciais erradas"
